tools/serverpod_cli/lib/src/analyzer/models/converter/converter.dart

```diff
@@ -26,6 +26,10 @@ T unsafeConvertToEnum<T extends Enum>({
   required dynamic value,
   required List<T> enumValues,
 }) {
+  if (value is! String) {
+    throw ArgumentError('Not a string: $value', 'value');
+  }
```

Comment:
Question: @Isakdl - How will this propagate to the user?

Reply:
This would have thrown before as well so there is no change to the runtime of this codepath.